temporalcli/commandsgen/commands.yml

```
@@ -213,6 +213,20 @@ commands:
       ```
     option-sets:
       - client
+    docs:
+      description: |
```

```suggestion
      description: >-
```

This will do the proper thing of making it a single line